src/routes/settings.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  // Components
+  import Settings from '../components/templates/Settings.svelte'
+</script>
+
+<style>
+</style>
+
+<svelte:head>
+  <title>FemAlert | Home</title>
+</svelte:head>
```

Comment:
It is even better to put the following here: **Instellingen | FemAlert** (in the title)